### DISCUSSION TITLE
Source code for Fluent UI implementation adaptivecards-fluentui

### DISCUSSION THREAD

**Discussion by GKersten** [microsoft/AdaptiveCards#6891] [General]
In our Angular Application we are using Adaptive Cards, we created custom renderers to render the components in Material Design. We took a lot of inspiration from the ["Fabric UI" implementation](https://github.com/microsoft/AdaptiveCards/tree/8c913fe9c2219653383d8f0982013d390311cd05/source/nodejs/adaptivecards-fabric), it was great resource to see how the custom renderers were implemented. We found that the [documentation for extensibility](https://docs.microsoft.com/en-us/adaptive-cards/sdk/rendering-cards/javascript/extensibility) was coming short in some places.

We noticed that the `adaptivecards-fabric` is removed in favor of [`adaptivecards-fluentui`](https://www.npmjs.com/package/adaptivecards-fluentui):
![image](https://user-images.githubusercontent.com/5525173/149618422-326c30c0-9531-449f-8e08-559e13e124d6.png)

We would really appreciate having access to the source code of `adaptivecards-fluentui` to be able to learn how to implement custom renderers with the latest `adaptivecards` packages. Are there any plans for making this public any time soon?

**Main reason:**
We recently did an upgrade of the `adaptivecards` package, version `2.9.0` -> `2.10.0` and this introduced a bug, where we are unable to trigger the submit action. So far no luck yet in resolving this issue.

**Comment by matthidinger** [comment]
Hi @GKersten, the source code for that package can be found here in a branch. Eventually we want to bring it to main but haven't found time for that yet.

https://github.com/microsoft/AdaptiveCards/tree/mahiding/fluent-ui/source/nodejs/adaptivecards-fluentui

**Reply by salimchemes** [reply]
hi, any updates on this? thanks
